lib/ansiblelint/rules/PackageHasRetryRule.py

```diff
@@ -18,7 +18,7 @@ class PackageHasRetryRule(AnsibleLintRule):
     #   | sort | awk -F '/' \
     #   '/__|dpkg|_repo|_facts|_sub|_chan/{next} {split($NF, words, ".");
     #   print "\""words[1]"\","}'
-    package_modules = [
+    _package_modules = [
```

Comment:
Why do you think this change is related to the PR purpose? I'd not want to see non-related changes making PR non-atomic and addressing multiple things.